### PR TITLE
fixed broken link pointing to localhost

### PR DESCRIPTION
**Pull request by onyiny-ang**

### CODE REVIEW

**Review by onyiny-ang — COMMENTED** [review]
same problem at the end too

File: pacman-nodejs-app/docs/pacman-nodejs-app-gke-migration.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Follow these steps to [push the Pac-Man container image to your Google Cloud Con
 
 ## Create Kubernetes Clusters
 
-Follow [these instructions](http://localhost:6419/docs/kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)
+Follow [these instructions](https://github.com/font/k8s-example-apps/blob/master/pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)
```
Comment (font) [inline pacman-nodejs-app/docs/pacman-nodejs-app-gke-migration.md]: No need to have the entire link. Just use `Follow [these instructions](kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)`

File: pacman-nodejs-app/docs/pacman-nodejs-app-gke-migration.md [hunk 2]
```diff
@@ -421,5 +421,5 @@ kubectl --context=gke_${GCP_PROJECT}_us-central1-b_gce-us-central1 \
 
 ### Remove Kubernetes Clusters
 
-Follow [these instructions](http://localhost:6419/docs/kubernetes-cluster-gke-federation.md#delete-kubernetes-clusters)
+Follow [these instructions](ttps://github.com/font/k8s-example-apps/blob/master/pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md#delete-kubernetes-clusters)
```
Comment (font) [inline pacman-nodejs-app/docs/pacman-nodejs-app-gke-migration.md]: Same comment as above.

**Review by font — CHANGES_REQUESTED** [review]
Thanks for the fixes!

**Review by font — APPROVED** [review]
I went ahead and made the changes.

**Pull request merged**